src/hubmap_translation/addl_index_transformations/portal/config.yaml

```diff
@@ -6,6 +6,20 @@ settings:
 mappings:
   date_detection: False
   dynamic_templates:
+    -
+      transposition_kit_number:
+        path_match: "*.transposition_kit_number"
```

Comment:
Currently, this is the only special case that we have. Without this mapping the full reindex process completes with severlal missing entities. We have observed that this property is typically a number, however, there are multiple instances of it containing "na". I believe this property exists in an embedded metadata field so that could be the root of this.

Reply:
Could new special cases be introduced as new documents are indexed or document index order changes?